build.xml

```diff
@@ -1247,6 +1340,7 @@
       <jvmarg value="-Dcassandra.ring_delay_ms=1000"/>
       <jvmarg value="-Dcassandra.tolerate_sstable_size=true"/>
       <jvmarg value="-Dcassandra.skip_sync=true" />
+      <jvmarg value="-Dnet.bytebuddy.experimental=true"/>
```

Comment:
I don't think you need to add it individually to each test configuration, to IntellliJ files, etc. - just add it to jdk22 props which are applied everywhere